packages/sdk/src/credential/teamsUserCredential.browser.ts

```diff
@@ -219,73 +267,27 @@ export class TeamsUserCredential implements TokenCredential {
   }
 
-  private async exchangeAccessTokenFromSimpleAuthServer(
-    scopesStr: string,
-    authCodeResult: AuthCodeResult
-  ): Promise<void> {
-    const axiosInstance: AxiosInstance = await this.getAxiosInstance();
-
-    let retryCount = 0;
-    while (true) {
-      try {
-        const response = await axiosInstance.post("/auth/token", {
-          scope: scopesStr,
-          code: authCodeResult.code,
-          code_verifier: authCodeResult.codeVerifier,
-          redirect_uri: authCodeResult.redirectUri,
-          grant_type: GrantType.authCode,
-        });
-
-        const tokenResult: AccessTokenResult = response.data;
-        const key = await this.getAccessTokenCacheKey(scopesStr);
-        // Important: tokens are stored in sessionStorage, read more here: https://aka.ms/teamsfx-session-storage-notice
-        this.setTokenCache(key, {
-          token: tokenResult.access_token,
-          expiresOnTimestamp: tokenResult.expires_on,
-        });
-        return;
-      } catch (err: any) {
-        if (err.response?.data?.type && err.response.data.type === "AadUiRequiredException") {
-          internalLogger.warn("Exchange access token failed, retry...");
-          if (retryCount < maxRetryCount) {
-            await this.sleep(retryTimeSpanInMillisecond);
-            retryCount++;
-            continue;
-          }
-        }
-        throw err;
-      }
-    }
-  }
-
-  /**
-   * Get access token cache from authentication server
-   * @returns Access token
-   */
-  private async getAndCacheAccessTokenFromSimpleAuthServer(
-    scopesStr: string
-  ): Promise<AccessToken> {
-    try {
-      internalLogger.verbose(
-        "Get access token from authentication server with scopes: " + scopesStr
-      );
-      const axiosInstance: AxiosInstance = await this.getAxiosInstance();
-      const response = await axiosInstance.post("/auth/token", {
-        scope: scopesStr,
-        grant_type: GrantType.ssoToken,
-      });
+  private async init(): Promise<void> {
+    microsoftTeams.initialize();
 
-      const accessTokenResult: AccessTokenResult = response.data;
-      const accessToken: AccessToken = {
-        token: accessTokenResult.access_token,
-        expiresOnTimestamp: accessTokenResult.expires_on,
-      };
-      const cacheKey = await this.getAccessTokenCacheKey(scopesStr);
-      this.setTokenCache(cacheKey, accessToken);
-      return accessToken;
-    } catch (err) {
-      throw this.generateAuthServerError(err);
-    }
+    const ssoToken = await this.getSSOToken();
+    const info = getTenantIdAndLoginHintFromSsoToken(ssoToken.token);
+    this.loginHint = info.loginHint;
+    this.tid = info.tid;
+
+    const msalConfig = {
+      auth: {
+        clientId: this.config.clientId!,
+        authority: `https://login.microsoftonline.com/${this.tid}`,
+      },
+      cache: {
+        cacheLocation: "sessionStorage",
+        storeAuthStateInCookie: false,
```

Comment:
what's the purpose of set this config?

Reply:
According to https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/configuration.md#cache-config-options, all the configs are same as default, thus remove them all.